Code/Tools/ProjectManager/Source/GemCatalog/GemItemDelegate.cpp

```diff
@@ -157,19 +156,29 @@ namespace O3DE::ProjectManager
         painter->drawText(gemCreatorRect, Qt::TextSingleLine, gemCreator);
 
         // Gem summary
-        const QStringList featureTags = GemModel::GetFeatures(modelIndex);
-        const bool hasTags = !featureTags.isEmpty();
-        const QString summary = GemModel::GetSummary(modelIndex);
+        const bool hasTags = !gemInfo.m_features.isEmpty();
         const QRect summaryRect = CalcSummaryRect(contentRect, hasTags);
-        DrawText(summary, painter, summaryRect, standardFont);
+        DrawText(gemInfo.m_summary, painter, summaryRect, standardFont);
+
+        // Gem Version
+        // include the version in the name if it isn't unknown
+        if (!gemInfo.m_version.isEmpty() && !gemInfo.m_version.contains("unknown", Qt::CaseInsensitive))
+        {
+            QPair<int, int> versionXBounds = CalcColumnXBounds(HeaderOrder::Version);
+            QRect gemVersionRect{ versionXBounds.first, contentRect.top(), versionXBounds.second - versionXBounds.first, contentRect.height() };
+            painter->setFont(standardFont);
+            gemVersionRect = painter->boundingRect(gemVersionRect, Qt::TextWordWrap | Qt::AlignRight | Qt::AlignVCenter, gemInfo.m_version);
+            painter->drawText(gemVersionRect, Qt::TextWordWrap | Qt::AlignRight | Qt::AlignVCenter, gemInfo.m_version);
```

Comment:
Does the word wrap occur only after each `.` of the semantic version?

Reply:
At word boundaries: https://doc.qt.io/qt-5/qt.html#TextFlag-enum